### PR TITLE
Seo optim

### PR DESCRIPTION
**Pull request by danylboiko95**

### CODE REVIEW

File: Examples/src/components/App.tsx [hunk 1]
```diff
@@ -23,7 +23,18 @@ const useStyles = makeStyles(
         },
         drawer: {
```
Comment (klishevich) [inline Examples/src/components/App.tsx:24]: shall we rename to drawerDesktop

File: Examples/src/components/App.tsx [hunk 2]
```diff
@@ -23,7 +23,18 @@ const useStyles = makeStyles(
...
             width: drawerWidth,
-            flexShrink: 0
+            flexShrink: 0,
+            [theme.breakpoints.down("md")]: {
+                display: "none"
+            }
+        },
+        drawer1: {
```
Comment (klishevich) [inline Examples/src/components/App.tsx:31]: and this one to drawerMobile

File: Examples/src/components/Examples/Charts2D/ChartAnnotations/AnnotationsAreEasy/GENERATED_SRC.ts [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
+import {BoxAnnotation} from "scichart/Charting/Visuals/Annotations/BoxAnnotation";
+import {CustomAnnotation} from "scichart/Charting/Visuals/Annotations/CustomAnnotation";
+import {TextAnnotation} from "scichart/Charting/Visuals/Annotations/TextAnnotation";
+import {EHorizontalAnchorPoint, EVerticalAnchorPoint} from "scichart/types/AnchorPoint";
+import {ECoordinateMode} from "scichart/Charting/Visuals/Annotations/AnnotationBase";
+import {EAnnotationLayer} from "scichart/Charting/Visuals/Annotations/IAnnotation";
+import {HorizontalLineAnnotation} from "scichart/Charting/Visuals/Annotations/HorizontalLineAnnotation";
+import {ELabelPlacement} from "scichart/types/LabelPlacement";
+import {VerticalLineAnnotation} from "scichart/Charting/Visuals/Annotations/VerticalLineAnnotation";
+import {AxisMarkerAnnotation} from "scichart/Charting/Visuals/Annotations/AxisMarkerAnnotation";
```
Comment (klishevich) [inline Examples/src/components/Examples/Charts2D/ChartAnnotations/AnnotationsAreEasy/GENERATED_SRC.ts:17]: please run prettier

File: Examples/src/components/Examples/Charts2D/ChartAnnotations/EditableAnnotations/GENERATED_SRC.ts [hunk 4]
```diff
@@ -100,8 +100,8 @@ export const drawExample = async () => {
             isEditable: true
         }),
         new TextAnnotation({
-            x1: 4,
-            y1: 2,
```
Comment (klishevich) [inline Examples/src/components/Examples/Charts2D/ChartAnnotations/EditableAnnotations/GENERATED_SRC.ts:104]: what are these hardcoded values?

File: Examples/src/components/Examples/Charts2D/ChartAnnotations/AnnotationsAreEasy/GENERATED_SRC.ts (hunk 3 above)
Comment (klishevich) [reply Examples/src/components/Examples/Charts2D/ChartAnnotations/AnnotationsAreEasy/GENERATED_SRC.ts:17]: and regenerate ExamplesSrc

**Review by klishevich — CHANGES_REQUESTED** [review]

File: Examples/src/components/Examples/Charts2D/ChartAnnotations/EditableAnnotations/GENERATED_SRC.ts (hunk 4 above)
Comment (danylboiko95) [reply Examples/src/components/Examples/Charts2D/ChartAnnotations/EditableAnnotations/GENERATED_SRC.ts:104]: I changed it for a better view on mobile devices

**Review by klishevich — APPROVED** [review]

**Pull request merged**